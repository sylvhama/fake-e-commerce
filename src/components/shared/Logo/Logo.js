import React from 'react';

export default props => (
  <svg xmlns="http://www.w3.org/2000/svg" width="182" height="50">
    <rect x="2" y="2" width="178" height="46" fill="#dedede" />
    <text
      x="50%"
      y="50%"
      fontSize="18"
      textAnchor="middle"
      alignmentBaseline="middle"
      fill="#555555"
    >
      Logo SVG
    </text>
  </svg>
);
