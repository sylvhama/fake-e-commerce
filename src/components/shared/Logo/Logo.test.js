import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../tests/helpers';

import Logo from './Logo';

test('Logo renders correctly', () => {
  const rendered = renderWithTheme(<Logo>Logo</Logo>).toJSON();
  expect(rendered).toMatchSnapshot();
});
