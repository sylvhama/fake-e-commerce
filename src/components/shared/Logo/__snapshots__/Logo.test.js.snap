// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<svg
  height="50"
  width="182"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    fill="#dedede"
    height="46"
    width="178"
    x="2"
    y="2"
  />
  <text
    alignmentBaseline="middle"
    fill="#555555"
    fontSize="18"
    textAnchor="middle"
    x="50%"
    y="50%"
  >
    Logo SVG
  </text>
</svg>
`;
